WordPress/src/main/res/drawable/bg_colorful_dots_large.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="696dp"
+    android:height="157dp"
+    android:viewportWidth="696"
+    android:viewportHeight="157">
+  <path
+      android:pathData="M622.168,17.431m-2.518,0a2.518,2.518 0,1 1,5.036 0a2.518,2.518 0,1 1,-5.036 0"
+      android:fillColor="#D48FC8"/>
```

Comment:
@ashiagr same as with bg_colorful_dots.xml? wdyt?

Reply:
Replaced here: 463e54e
Couldn't find colors for red and blue dots. Will check with Shakti and update.

Reply:
Replaced red and blue dot colors in ccc1f86